Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
```

Comment:
That means those formulae will need changing too :(

Here's another question. Let's say we go with the old npm for now. What happens when someone tries to upgrade npm in place at that point?